src/components/pages/Publications/PublicationsConstants.ts

```diff
@@ -421,6 +421,51 @@ export const listOfMediaPublications: PublicationProps[] = [
 
 // Note: for biblio digests, date = date published
 export const listOfBiblioDigests: BiblioDigestProps[] = [
+    {
+        url: "https://drive.google.com/file/d/1vJ1SYAmuDE-Gpx6wlWIV6gY_tploGujn/view",
+        sourcesAdded: 541,
```

Comment:
Should be 41 (instead of 541).

Reply:
Thanks, Himanshu! Highlighted a typo above; all else looks good.

As a consideration for a future update, our data users at WHO asked if we could put the CSV files of Biblios on the website too.